src/singletons/Settings.hpp

```diff
@@ -126,6 +126,7 @@ class Settings : public ABSettings, public ConcurrentSettings
     BoolSetting showParts = {"/behaviour/showParts", false};
     FloatSetting mouseScrollMultiplier = {"/behaviour/mouseScrollMultiplier",
                                           1.0};
+    BoolSetting autoCloseUserPopup = {"/behaviour/autoCloseUserPopup", false};
```

Comment:
Personally, I think the setting should be `true` by default since that is the behavior in the current stable version (2.1.7).

Reply:
Changed to be `true` by default